### PR TITLE
Adding onAfterElementRendered hook

### PR DESCRIPTION
**Pull request by meirotstein**

### CODE REVIEW

**Comment by meirotstein** [comment]
A PR for #19

File: README.md [hunk 1]
```diff
@@ -69,9 +69,45 @@ Usage
...
+| `onAfterElementRendered` | Function | A callback function that will be invoked after each render on an element, it allows to customize the result DOM element. see the following for exact signature and usage example of the callback function. |  |
+
+Example callback function for `onAfterElementRendered`:
+```js
+/**
+* @param {HTMLElement} element html element that was rendered by json-pollock
+* @param {Object} template SC template of this element  
+* @returns {HTMLElement} manipulated html element
+*/
+(element, template) => {
```
Comment (eliranmal) [inline README.md]: perhaps it's better to update this to a named function, for better readability:

```js
function onAfterElementRendered(element, template) {
  // ...
}
// or:
const onAfterElementRendered = (element, template) => {
  // ...
}
```

File: README.md [hunk 2]
```diff
@@ -69,9 +69,45 @@ Usage
...
+    // maniplate elements:
+    switch (template.type) {
+      case 'text':
```
Comment (eliranmal) [inline README.md]: API users should not use bare strings; `template.type`s should be exported publicly as constants, e.g. `JsonPollock.TEMPLATE_TYPE.TEXT`.

do we have such thing?

Comment (meirotstein) [reply README.md]: Nope, good point. I'll add it.

File: README.md (hunk 1 above)
Comment (meirotstein) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (meirotstein) [reply README.md]: done

**Pull request merged**